README.md

```diff
@@ -30,6 +30,7 @@ For detailed changes on the `ingress-nginx` helm chart, please check the followi
 
 | Ingress-nginx version | k8s supported version  | Alpine Version | Nginx Version |
 |-----------------------|-------------           |----------------|---------------|
+| v1.0.1                | 1.22, 1.21, 1.20, 1.19 | 3.13.5         |  1.19.9       |
```

Comment:
Add also v0.49.1 here as we are going to release it together with this one :)

Also, Alpine now is 3.14.2

Reply:
done